### PR TITLE
[TE] frontend - Translate granularity dropdown options to human-readable format

### PR DESCRIPTION
**Pull request by ttbach**
- Granularity dropdown values are now grammatically correct
- Default compare by mode is changed to "split"

### CODE REVIEW

File: thirdeye/thirdeye-frontend/app/pods/components/rootcause-chart-toolbar/component.js [hunk 1]
```diff
@@ -42,12 +65,31 @@ export default Component.extend({
...
+    let selected = '';
+
+    // Find the key in granularityMapping whose value matches the granularity
+    keys.some(key => {
+      const value = this.granularityMapping[key];
+      if (value === get(this, 'granularity')) {
+        selected = key;
+      }
+    });
+
```
Comment (ttbach) [inline thirdeye/thirdeye-frontend/app/pods/components/rootcause-chart-toolbar/component.js]: @justYves Yeah I was itching about that. Should I extract the translation logic here into a util?

Comment (apucher) [reply thirdeye/thirdeye-frontend/app/pods/components/rootcause-chart-toolbar/component.js]: @ttbach imo the "correct" way would be to make this a two-sided map

Comment (ttbach) [reply thirdeye/thirdeye-frontend/app/pods/components/rootcause-chart-toolbar/component.js]: I agree. The only thing is there's no bi-directional map in javascript 😞

Comment (igbopie) [reply thirdeye/thirdeye-frontend/app/pods/components/rootcause-chart-toolbar/component.js]: how about this?
`
const granularityValue = get(this, 'granularity');
const selected = keys.find(key => granularityValue === this.granularityMapping[key])
`

Comment (ttbach) [reply thirdeye/thirdeye-frontend/app/pods/components/rootcause-chart-toolbar/component.js]: Ooh i like that. Thanks for the suggestion! @igbopie.

**Pull request merged**